funcs.sh

```diff
@@ -27,6 +27,12 @@ function disable_postinstall_vendor_expose {
 	fi
 }
 
+# Copied from CWP, some sites still use the bespoke "private": "true" flag which redirects URLs to internal SSH.
```

Comment:
Can we maybe look to remove this from non-legacy builds entirely? I am not a fan of manipulating codebase on deployment to try and solve developer problems (e.g. Using https:// instead of ssh://)

Reply:
It's a feature, and one that 10-20 stacks rely on. I would prefer to fight this battle some other day.

Reply:
Actually, maybe we can drop it on composer v2 though, since nobody switched yet and it requires a rebuild anyway. Would just need to confirm though it can still be done without this feature - presumably one could point to ssh source directly.

Reply:
It is 10x harder to remove something after its already added IMO. If we leave it in legacy builds, then people who rely on it can stay on legacy, and if they want to migrate to more advance builds then they will need to sort out these issues.

Reply:
Would prefer to drop in v2 if possible. Then its a pathway out.

Reply:
They can't stay on legacy (which is "build on dash"), it's insecure and I want to get rid of it -  so I don't want any obstacles there. So will move this PR to 1.3 branch (composer v1), but not have it in v2, and document it's backwardly incompatible.